src/main/java/zd/zero/waifu/motivator/plugin/settings/WaifuMotivatorSettingsPage.java

```diff
@@ -272,11 +272,6 @@ public boolean isCellEditable( Integer info) {
 
         exitCodeTable.setSelectionMode(ListSelectionModel.SINGLE_SELECTION);
 
-        new CellTooltipManager(ApplicationService.INSTANCE).
-            withCellComponentProvider( CellComponentProvider.forTable( exitCodeTable )).
-            installOn( exitCodeTable );
-
-
```

Comment:
So I've been trying to figure out what this statement does is it just adds support for a tooltip mouse hover of the table right? and we don't have a tooltip, it's fine to just remove this.